### ISSUE TITLE
Opening App (even demo project) LANDSCAPE on ipad

### ISSUE THREAD

**Issue by ghost** [ECSlidingViewController/ECSlidingViewController#131]
Opening App (even demo project) LANDSCAPE on ipad, navigating by button to right viewController placed underneath initial loading Controller via:
    [self.slidingViewController anchorTopViewTo:ECLeft];

When loaded page works, but cannot navigate back to initialController. Only gesture works, this is coupled with the issue of the initialController (tableView) being active also. User can interact.

With Demo provided in zip i HAVE been able to recreate this, as at first i thought it was something i had done. This also happens both ipad simulator and on device

I am navigating to a search in a TableView embed in Navigation Controller.

NOTE: In both my project and demo, if you open in portrait it's fine. and if you load landscape and then were to change orientation the Peek amount of the initialController is no longer active and touching anywhere, as you expect, returns you to initialController.

To recap on issue:
Open App landscape.
Top UIBarButton  to TableView in Navigation Controller
Remaining Landscape
User not able to navigate back 
InitialController along with loaded controller both active.

Changing orientation then removes problem.

From what i understand ECslinding takes a screenshot of the topViewController and uses this as the peek? It seems as though this part of the method is not happening?

Thanks in advance for your help.
